code/modules/events/abductor.dm

```diff
@@ -0,0 +1,61 @@
+/datum/round_event_control/abductor
+	name = "Abductors"
+	typepath = /datum/round_event/abductor
+	weight = 5
+	max_occurrences = 1
+
+	earliest_start = 180000 // 30 min
+
+	gamemode_blacklist = list("nuclear","wizard","cult","gang","revolution","abduction","blob")
```

Comment:
I don't see why ANY game modes should be excluded, the only reason this var existed was to keep random blobs out of blob mode. Abductors are extremely passive antags that can hang out in pretty much any round type. If you exclude a bunch it just allows for meta about the roundtype if you see an abductor.